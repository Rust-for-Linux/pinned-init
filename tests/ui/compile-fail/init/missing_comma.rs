use pin_init::*;

#[pin_data]
struct Foo {
    a: Bar,
    b: Bar,
    c: Bar,
}

struct Bar;

fn main() {
    let _ = pin_init!(Foo {
        a: Bar,
        b: Bar
        c: Bar,
    });
}
