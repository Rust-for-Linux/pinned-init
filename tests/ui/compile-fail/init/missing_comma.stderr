error: expected `,`
  --> tests/ui/compile-fail/init/missing_comma.rs:16:9
   |
16 |         c: Bar,
   |         ^
